.github/workflows/release.yml

```diff
@@ -48,28 +52,20 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: ${{ env.POETRY_VERSION }}-python-${{ matrix.python }}
         uses: docker/build-push-action@v2
+        if: ${{ matrix.variant == "buster" }}
         with:
           push: true
           tags: ${{ env.GHCR_IMAGE_REPOSITORY }}:${{ env.POETRY_VERSION }}-python-${{ matrix.python }}
           build-args: |
             POETRY_VERSION=${{ env.POETRY_VERSION }}
             PYTHON_VERSION=${{ matrix.python }}
-            VARIANT=buster
```

Comment:
I think the CI, should not assume the default `VARIANT` in the Dockerfile.

Reply:
This is not that easy to do, with `include` it's easier to do it and all defaults are available in one place.